spec/index.html

```diff
@@ -14,6 +14,7 @@ <h1>Atomics.pause ( [ _iterationNumber_ ] )</h1>
     1. If _iterationNumber_ is not *undefined*, then
       1. If _iterationNumber_ is not an integral Number, throw a *TypeError* exception.
       1. If ℝ(_iterationNumber_) &lt; 0, throw a *RangeError* exception.
+    1. If an ECMAScript implementation is running on a processor that supports an instruction to signal that the current executing code is in a spin-wait loop, it may perform that instruction.
```

Comment:
I would be a little stronger and more general maybe?

```suggestion
    1. If the execution environment of the ECMAScript implementation supports a signal that the current executing code is in a spin-wait loop, for example via a hardware instruction, the implementation should send that signal.
```

Reply:
s/should/must, then?

Reply:
We don't usually should/must. Why not "... instruction, send that signal"?